docs/decisions/0009-pylint-assertions.rst

```diff
@@ -0,0 +1,50 @@
+Pylint Assertions Standardization
+---------------------------------
+
+Status
+======
+
+Proposed
```

Comment:
I came here after finding large conflicts during a rebase due to the following PR: https://github.com/edx/edx-platform/pull/26308
This ADR is already being implemented, before the discussion has actually taken place (as mentioned in the PR description). Is it still time to revert this ADR? If not, this should be "Accepted".